tests/integration/pilot/outboundtrafficpolicy/helper.go

```diff
@@ -283,20 +283,24 @@ func RunExternalRequestTest(expected map[string][]string, t *testing.T) {
 				portName string
 				host     string
 				gateway  bool
+				scheme   scheme.Instance
 			}{
 				{
 					name:     "HTTP Traffic",
 					portName: "http",
+					scheme:   scheme.HTTP,
 				},
 				{
 					name:     "HTTPS Traffic",
 					portName: "https",
+					scheme:   scheme.TCP,
```

Comment:
Why is this TCP? We should be sending HTTP(s) requests here, not the "Hello World" message?

Reply:
Actually I don't even get how this is passing tests. How does a TCP request generate a 200 response code?

Reply:
The echo server writes "StatusCode=200" to hack the client.

Reply:
ok that makes sense. Why are we sending TCP here though?

Reply:
Yes, the client sends "HelloWorld" and the server replies
```
StatusCode=200
HelloWorld
```

Reply:
The server echo the message from client with the status prefix

Reply:
But why is this TCP? The test is supposed to be testing HTTPS.

Reply:
Let me make some change for HTTPs. The docker and kube test logic is a little bit different.